### PR TITLE
Connect now playing action bar

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

* Const existing action bar
* Move cast logic to common store to connect cast button

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Simulator

https://user-images.githubusercontent.com/2731362/155643003-015b311a-4daa-489c-b5bd-54a8f0f6fab8.mp4

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

Mobile release / sentry mobile crashes, etc.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-aud-1413-2

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-aud-1413-2

**Review by dylanjeffers — APPROVED** [review]
looks great!

File: packages/web/src/common/store/cast/sagas.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { takeEvery } from 'redux-saga/effects'
+
+import { CAST_METHOD, updateMethod } from './slice'
+
+function* watchUpdateCastMethod() {
+  yield takeEvery(updateMethod.type, function* (
+    action: ReturnType<typeof updateMethod>
+  ) {
+    const { method } = action.payload
+    window.localStorage.setItem(CAST_METHOD, method)
```
[the remark this answers is not in the document]
Comment (raymondjacobson) [reply packages/web/src/common/store/cast/sagas.ts:10]: woops, meant to swap this. thanks! will address

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-aud-1413-2

File: packages/web/src/common/store/cast/sagas.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (raymondjacobson) [reply packages/web/src/common/store/cast/sagas.ts:10]: follow up PR

**Pull request merged**